pkg/application/application.go

```diff
@@ -52,7 +49,7 @@ func List(client *occlient.Client) ([]config.ApplicationInfo, error) {
 // Shows also empty applications (empty applications are those that are just
 // mentioned in config file but don't have any object associated with it on cluster).
 func ListInProject(client *occlient.Client, project string) ([]config.ApplicationInfo, error) {
```

Comment:
We are already passing `client` that already has namespace set, we shouldn't need pass project here again